### ISSUE TITLE
[BUG]When using Set-PnPPageTextPart update text with a img link, the image was disappeared after user edit the page from SPO UI. 

### ISSUE THREAD

**Issue by frankchen76** [pnp/powershell#1733]
I used the below PS cmdlet to update a page which has a Text web part, the content was updated correctly. However, when an user updated that changed page from SPO page edit view and save it. the image was gone. 

```PowerShell
Set-PnPPageTextPart -Page PSTest1 -InstanceId 64149e67-f0d8-48fb-abfe-34b46aa28f3c -Text "<strong>MyText</strong><img src='/sites/FrankCommunication1/SiteAssets/SitePages/Event/48279-car-500x300.jpg'/>"
```

### Expected behavior 
the end user can update the page from SPO UI

### Actual behavior 
After running the PS to insert the img tag inside the page, when end user updated the page and save it from SPO UI, the image was disappeared. 

### Steps to reproduce behavior
Please see image below
![2022-03-29_22-48-51](https://user-images.githubusercontent.com/9618662/160760611-401c5547-04de-476f-9c12-870ef7625e5c.gif)

### What is the version of the Cmdlet module you are running?
1.9.0

### Which operating system/environment are you running PnP PowerShell on?
- [X] Windows
- [ ] Linux
- [ ] MacOS
- [ ] Azure Cloud Shell
- [ ] Azure Functions
- [ ] Other : please specify

**Comment by frankchen76** [comment]
@khamphou , thanks for your note. I realized that when I added a image, it became to separate web part which also provides additional functions like resize, zoom. just wondering if there is a way to add that programmatically?

**Comment by frankchen76** [comment]
It seems like the PnP.Framework used by PnP Powershell included inline web part logic. but the PnP.PowerShell (v1.10) which I used doesn't include that feature yet. So the solution is to interact with object directly like below. hopefully the new PnP.PowerShell version will include that. Thanks
```
$p = Get-PnPPage PSTest1
$wpText = $p.NewTextPart()
$html = $p.GetInlineImage($wpText, "/sites/FrankCommunication1/SiteAssets/SitePages/Event/48279-car-500x300.jpg")
$text = "<strong>MyText</strong>" + $html
$wpText.Text = $text
$p.AddControl($wpText, $p.Sections[0].Columns[0])
$p.Save()

```

**Comment by khamphou** [comment]
Thank you for the updates.
Is this new method available for pnp powershell nightly build 1.10.13 ?

the code you mentionned was tested and worked ? or just a portable version of the pnp core c# code ?
